Model/Transaction.php

```diff
@@ -96,6 +103,8 @@ public function __construct(
         $this->logger = $logger->setFilename(TaxjarConfig::TAXJAR_TRANSACTIONS_LOG);
         $this->objectManager = $objectManager;
         $this->helper = $helper;
+        $this->taxjarConfig = $taxjarConfig;
+        $this->apiKey = $this->taxjarConfig->getApiKey();
```

Comment:
@ryanreeves-taxjar Do we use the API token anywhere in this class or through the inherited `Order` and `Refund` classes? Otherwise can we drop this code in `Model/Transaction.php`?